.github/workflows/ci.yml

```diff
@@ -48,9 +48,6 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Test
-        run: npm run test
-
```

Comment:
Note: `npm run test` is running `build` + `test:e2e`. Both of these steps are already executed by this workflow.